hydra-node/test/Hydra/Chain/Direct/Contract/Close.hs

```diff
@@ -275,10 +325,18 @@ genCloseMutation (tx, _utxo) =
     , SomeMutation (Just $ toErrorCode ChangedParameters) MutateHeadIdInOutput <$> do
         otherHeadId <- toPlutusCurrencySymbol . headPolicyId <$> arbitrary `suchThat` (/= Fixture.testSeedInput)
         pure $ ChangeOutput 0 $ changeHeadOutputDatum (replaceHeadId otherHeadId) headTxOut
-    , SomeMutation Nothing MutateRequiredSigner <$> do
-        newSigner <- verificationKeyHash <$> genVerificationKey
+    , SomeMutation (Just $ toErrorCode SignerIsNotAParticipant) MutateRequiredSigner <$> do
+        newSigner <- verificationKeyHash <$> genVerificationKey `suchThat` (/= somePartyCardanoVerificationKey)
         pure $ ChangeRequiredSigners [newSigner]
-    , SomeMutation Nothing MutateCloseUTxOHash . ChangeOutput 0 <$> mutateCloseUTxOHash
+    , SomeMutation (Just $ toErrorCode NoSigners) MutateNoRequiredSigner <$> do
+        pure $ ChangeRequiredSigners []
+    , SomeMutation (Just $ toErrorCode TooManySigners) MutateMultipleRequiredSigner <$> do
```

Comment:
I would explain why it is a problem to have more than one signer for the transaction.

Reply:
already answered here https://github.com/input-output-hk/hydra/pull/719#discussion_r1129632155